### ISSUE TITLE
Small Tablet - Display 2-Pages (in landscape mode)

### ISSUE THREAD

**Issue by jbhorsewriter7** [futurepress/epub.js#348]
I am using a small mobile device with the dimensions 854px x 508px.
It also does this on an iPad mini - one page instead of two.
Is there any way that I can display 2 pages on this device (when it is in landscape) instead of just one.
I have tried modifying the css - but it looks like the javascript might be controlling things as well...

**Comment by fchasen** [comment]
Yep, just set `minSpreadWidth` to something lower.

```javascript
var book = ePub('path/to/book.epub', { minSpreadWidth: 768 });
```

**Comment by jbhorsewriter7** [comment]
This worked wonderfully - I set minSpreadWidth to 500 (along with my css changes).

Now I am having one issue.  When I change to a different chapter then this variable is missing or overwritten (just on the iPad) and the width of my pages are full (even though the divider is still visible).

On my computer and in Android this doesn't seem to happen...

**Comment by jbhorsewriter7** [comment]
Does anyone have any ideas how I might go about debugging this?

Is there a way to send the minSpreadWidth again - or a way that it could be being lost... or any custom code for iOS?

**Comment by fchasen** [comment]
Are you getting any JS errors? That shouldn't change when you switch chapters.
First thing to check is if `book.settings.minSpreadWidth` still is what you set it is after a chapter change.

You could also try `book.minSpreadWidth(768)` and see if that makes a difference.

**Comment by jbhorsewriter7** [comment]
It looks like this is from something that I modified in the CSS:

@media only screen and (max-width: 1040px) 

If I change the 1040px to 800px then it makes the iPad flip... so I am going to try ADDING an @media for 800px.

**Comment by jbhorsewriter7** [comment]
Well... it looks like there is something special that is just running on the iPad.  It looks like there is a check on the width that is being run that forces the width to be the full width.  Looks very strange as the width of the page jumps from being small to being wider than what is displayed.

I am very confused...

**Comment by jbhorsewriter7** [comment]
I don't think it has to do with the minSpreadWidth as I went in and replaced the default value in the JavaScript and everywhere else I could find it.  But I still might have missed something...

**Comment by jbhorsewriter7** [comment]
It looks like the resize listener is being triggered when the next chapter is loaded.  I am still working to figure out why.  But if the resizer did not run then it would not change the width like it has been.
